### DISCUSSION TITLE
(Intune) pending update prevents network connections and some GUI operations

### DISCUSSION THREAD

**Discussion by xerces8** [mozilla/policy-templates#967] [Q&A]
Hi!

While MS Intune performs/triggers an update of Firefox, the Firefox windows encounters these problems: network connections don't work any more , some GUI operations are missing or stuck

I described this in https://bugzilla.mozilla.org/show_bug.cgi?id=1788173

Did anyone here have similar issues? Is this related to Intune at all?

**Comment by mkaply** [comment]
The beast place to ask a question like this is our enterprise mailing list.

https://groups.google.com/a/mozilla.org/g/enterprise

But the bug is a good start as well. I think it's in the wrong place. I'll see if I can get someone to see it.
